File: 58-LengthOfLastWord/58-LengthOfLastWord.java
// Last updated: 5/25/2026, 11:21:14 AM
class Solution {
    public int lengthOfLastWord(String s) {
        String[] arr=s.split(" ");
        String str=arr[arr.length-1];
        char[] arr1=str.toCharArray();
        int a=arr1.length;
        return a;
    }
}

File: 58-LengthOfLastWord/58-LengthOfLastWord.py
# Last updated: 5/25/2026, 11:21:20 AM
class Solution(object):
    def lengthOfLastWord(self, s):
        s = s.strip()
        words = s.split()
        return len(words[-1])
        """
        :type s: str
        :rtype: int
        """
        